README.md

```diff
@@ -1,217 +1,263 @@
+- [uta](https://github.com/biocommons/uta): a database of transcripts and
+  alignments (localhost:5432)
 
-## Install PostgreSQL
+Check that the containers are running, by running:
 
-PostgreSQL is required for UTA. Ensure that PostgreSQL is installed on your system. You can download and install PostgreSQL from [here](https://www.postgresql.org/download/) or [install using Homebrew](#install-postgresql-using-homebrew).
+```shell
+$ docker ps
+CONTAINER ID        IMAGE                                    //  NAMES
+86e872ab0c69        biocommons/seqrepo-rest-service:latest   //  vrs-python_seqrepo-rest-service_1
+a40576b8cf1f        biocommons/uta:uta_20210129b              //  vrs-python_uta_1
+```
 
-There are [macOS docs](https://github.com/ga4gh/vrs-python/tree/2.x/docs/setup_help) for additional assistance.
+Depending on your network and host, the _first_ run is likely to take 5-15
+minutes in order to download and install data. Subsequent startups should be
+nearly instantaneous.
 
-### Install PostgreSQL using Homebrew
+You can test UTA and seqrepo installations like so:
 
-1. Install Homebrew: See the [documentation](https://docs.brew.sh/Installation) for how to install.
+```shell
+$ psql -XAt postgres://anonymous@localhost/uta -c 'select count(*) from transcript'
+249909
+```
 
-1. Update Homebrew: Make sure Homebrew is up-to-date by running:
+##### It doesn't work
 
-        brew update
+Here are some things to try.
 
-1. Install PostgreSQL: Use the following command to install [PostgreSQL](https://formulae.brew.sh/formula/postgresql@14#default):
+- Bring up one service at a time. For example, if you haven't download seqrepo
+  yet, you might see this:
 
-        brew install postgresql@14
+  ```shell
+  $ docker-compose up seqrepo-rest-service
+  Starting vrs-python_seqrepo-rest-service_1 ... done
+  Attaching to vrs-python_seqrepo-rest-service_1
+  seqrepo-rest-service_1  | 2022-07-26 15:59:59 seqrepo_rest_service.__main__[1] INFO Using seqrepo_dir='/usr/local/share/seqrepo/2021-01-29' from command line
+  ⋮
+  seqrepo-rest-service_1  | OSError: Unable to open SeqRepo directory /usr/local/share/seqrepo/2021-01-29
+  vrs-python_seqrepo-rest-service_1 exited with code 1
+  ```
 
-1. Start PostgreSQL: Once the installation is complete, you can start the PostgreSQL service using:
+## VRS-Python and VRS Version Correspondence
 
-        brew services start postgresql
+The ga4gh/vrs-python repo embeds the ga4gh/vrs repo as a git submodule for testing purposes.
+Each ga4gh.vrs package on PyPI embeds a particular version of VRS. The
+correspondences between the packages that are **currently maintained** may be summarized as:
 
-## Installing the VRS-Python package with pip
+| vrs-python branch | vrs-python tag/version | vrs branch | vrs version |
+| --- | --- | --- | --- |
+| [main](https://github.com/ga4gh/vrs-python/tree/main) _(default branch)_ | 2.x | [2.0-alpha](https://github.com/ga4gh/vrs/tree/2.0-alpha) | 2.x |
```

Comment:
The `main` vrs-python is NOT based on `2.0-alpha` anymore. It looks like the submodule was updated to some commit around March 6 and `2.0-alpha` is 2 months old at this point. I'm wondering if we should bother trying to keep a specific branch or commit in this list and instead keep it related to the `latest in 2.x` since that's what it should always be?